autosklearn/ensembles/ensemble_selection.py

```diff
@@ -16,15 +18,46 @@ def __init__(
         ensemble_size: int,
         task_type: int,
         metric: Scorer,
-        random_state: np.random.RandomState,
         bagging: bool = False,
         mode: str = 'fast',
+        random_state: Optional[Union[int, np.random.RandomState]] = None,
     ) -> None:
+        """ An ensemble of selected algorithms
+
+        Fitting an EnsembleSelection generates an ensemble from the the models
+        generated during the search process. Can be further used for prediction.
+
+        Parameters
+        ----------
+        task_type: int
+            An identifier indicating which task is being performed.
+        metric: Scorer
+            The metric used to evaluate the models
+        bagging: bool = False
+            Whether to use bagging in ensemble selection
+        mode: str in ['fast', 'slow'] = 'fast'
+            Which kind of ensemble generation to use
+            *   'slow' - The original method used in Rich Caruana's ensemble selection.
+            *   'fast' - A faster version of Rich Caruanas' ensemble selection.
+
+        random_state: Optional[int | RandomState] = None
+            The random_state used for ensemble selection.
+            *   None - Uses numpy's default RandomState object
+            *   int - Successive calls to fit will produce the same results
+            *   RandomState - Truely random, each call to fit will produce
+                              different results, even with the same object.
+        """
         self.ensemble_size = ensemble_size
         self.task_type = task_type
         self.metric = metric
         self.bagging = bagging
         self.mode = mode
+
+        # Behaviour similar to sklearn
+        #   int - Deteriministic with succesive calls to fit
+        #   RandomState - Successive calls to fit will produce differences
+        #   None - Uses numpmys global singleton RandomState
+        # https://scikit-learn.org/0.24/common_pitfalls.html#controlling-randomness
```

Comment:
Could you please link to stable instead of a specific version here?

Reply:
Good catch, I'll keep a note to look for that in the future.